sw.js

```diff
@@ -38,6 +38,7 @@ self.onmessage = event => {
       metadata[0] = evt.data.readableStream
     }
   } else {
+    port.postMessage({ userAborted: false })
```

Comment:
Probably would have called it just abort, cancel, aborted or cancelled. The stream can abort without user interaction like having a timed AbortController